app/controllers/admin/claim_checks_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Admin::ClaimChecksController < Admin::BaseAdminController
+  before_action :ensure_service_operator
+  before_action :load_claim
+  before_action :reject_checked_claims
+
+  def create
+    @claim.create_check!(checked_by: admin_session.user_id, result: params[:result])
```

Comment:
Can we require the `result` param. It's really easy to forget to pass it during specs.

Reply:
I think if we add the validations to the model then we shouldn't need to as this line will fail noisily with an exception if the result is missing.

Reply:
My point about this one is that we previously had an error case test passing because the page rendered an error code, even though the reason was because of missing parameters, rather than creating another check.

Reply:
If we were to do this, would it be with strong parameters?

Reply:
I bring it up because it _was_ happening until I did my last rebase. Happy to be overruled.

Reply:
Yep, the standard Rails way to do it would probably be `ActionController::Parameters#require`, which incidentally raises an exception if the required parameter isn't present!

Reply:
@erbridge do you have a suggestion for what the code to protect against false-positives might look like?

Reply:
@erbridge that spec feels a bit off to me anyway, is it testing that a support user cannot approve a claim, as in create a check, so would the 'you are not an approver' fail first? 

When I change the behaviour I see what I expect, forgetting the `result` param causes a `ActiveRecord::RecordInvalid` error with the validations.

Reply:
No suggestions for now, sorry.